crates/task-impls/src/transactions.rs

```diff
@@ -204,8 +227,15 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, Ver: StaticVersionType>
                     let membership_total_nodes = self.membership.total_nodes();
 
                     // Calculate the builder fee for the empty block
-                    let Some(builder_fee) = null_block::builder_fee(membership_total_nodes) else {
-                        error!("Failed to get builder fee");
+                    let Some(bid_fee) = null_block::builder_fee(membership_total_nodes) else {
+                        error!("Failed to get bid fee");
+                        return None;
+                    };
+
+                    let Some(sequencing_fee) =
+                        null_block::builder_fee(self.membership.total_nodes())
+                    else {
+                        error!("Failed to get sequencing fee");
```

Comment:
Minor nit: one uses `self.membership.total_nodes()`, while the other uses `membership_total_nodes`, but I think they're both exactly the same thing.

Perhaps: `let Some(null_fee) = null_block::builder_fee(self.membership.total_nodes()` then `let bid_fee = null_fee; let sequencing_fee = null_fee;`?

Reply:
Done